x/globalfee/ante/fee.go

```diff
@@ -164,21 +174,22 @@ func (mfd FeeDecorator) GetMaxTotalBypassMinFeeMsgGasUsage(ctx sdk.Context) (res
 
 // GetMinGasPrice returns a nodes's local minimum gas prices
 // fees given a gas limit
-func GetMinGasPrice(ctx sdk.Context, gasLimit int64) sdk.Coins {
+func GetMinGasPrice(ctx sdk.Context, gasLimit int64) sdk.DecCoins {
 	minGasPrices := ctx.MinGasPrices()
 	// special case: if minGasPrices=[], requiredFees=[]
 	if minGasPrices.IsZero() {
-		return sdk.Coins{}
+		return sdk.DecCoins{}
 	}
 
-	requiredFees := make(sdk.Coins, len(minGasPrices))
-	// Determine the required fees by multiplying each required minimum gas
-	// price by the gas limit, where fee = ceil(minGasPrice * gasLimit).
-	glDec := sdk.NewDec(gasLimit)
-	for i, gp := range minGasPrices {
-		fee := gp.Amount.Mul(glDec)
-		requiredFees[i] = sdk.NewCoin(gp.Denom, fee.Ceil().RoundInt())
-	}
+	ctx.Logger().Error("[DEBUG]:", "minGasPrices  is ", minGasPrices)
```

Comment:
Should probably change this to a ctx.Logger().Debug() if you want to leave it in